src/components/Register.js

```diff
@@ -27,28 +26,16 @@ export default function Register() {
             // return error 
             return setError("ERROR: Passwords do not match ")
         }
-        try {
-            // set error back to empty string 
-            setError("")
-            // set up loading state (only can create one account at the same time, cant keep clicking submit button)
-            setLoading(true)
-            // call register function from AuthContext
-            const { email, uid } = (await register(emailRef.current.value, passwordRef.current.value)).user
-            await postUser({
-                userid: uid,
-                useremail: email,
-                userphonenumber: phoneNumberRef.current.value
-            })
-            history.push("/profile")
 
-        }
-        catch {
-            // error message 
-            setError("ERROR: Failed to create account")
-        }
-
-        // after all handling is done 
-        setLoading(false)
+        // set error back to empty string 
+        setError("")
+        // set up loading state (only can create one account at the same time, cant keep clicking submit button)
+        setLoading(true)
+        // call register function from AuthContext
+        register(emailRef.current.value, passwordRef.current.value, phoneNumberRef.current.value)
+            .then(() => history.push("/profile"))
+            .catch(err => setError(`ERROR: ${err}`))
```

Comment:
this line returns: ERROR: Error: Password should be at least 6 characters
so get rid of "ERROR" and wherever "ERROR" is in other places change it to "Error" so uniform

Reply:
We will need to create a bug for this issue. Error messages will need to be updated.

Reply:
Bug created at #71